ggml/src/ggml.c

```diff
@@ -121,8 +143,14 @@ static int sched_yield (void) {
     return 0;
 }
 #else
+
+#ifndef __USE_GNU
+#define __USE_GNU
+#endif
```

Comment:
Is it really ok to define this macro manually? It should probably be `_GNU_SOURCE`, but that is already defined in the build scripts.

Reply:
Yeah, I think we had some compiler errors on Android otherwise.
Will double check.

Reply:
looks like we just need to update cmake to set that. 
will push the fix tomorrow (maybe new PR if you merge this one before that)